src/main/java/com/mg105/view_interfaces/Alert.java

```diff
@@ -0,0 +1,16 @@
+package com.mg105.view_interfaces;
```

Comment:
We are organizing our code by layer, `view_interfaces` is not one of the clean architecture layers. If this is only needed in the user interface (which as far as I can tell is true), it should just be in the `user_interface` package...?